system/Database/SQLite3/Connection.php

```diff
@@ -332,26 +332,53 @@ protected function _foreignKeyData(string $table): array
             return [];
         }
 
```

Comment:
Sqlite for some reason doesn't have an easy way to return the constraint name for foreign keys. The only thing you can do is look at the create table statement and see the name that was used.

The previous implementation returned some odd descriptions as constraint_name. The changes here build the index name as it would have been created with Forge. 

Throughout Forge and with these methods the constraint names have all been normalized. Each DBMS will use the same foreign key constraint names. This was not so before this PR. This made the tests quite a mess.